From 5455b039c75abff93b7cb0a2d2023a9a100e6286 Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Mon, 31 Aug 2015 09:33:38 +0300
Subject: [PATCH 13/15] imx_v7_android_defconfig: enable Wilink8 related
 switches

* Disable internal mac80211 and cfg80211 as we are building wilink8
  drivers using backport and loading as modules
* enable wlan and hci related switches
* enable bluetooth shared transport

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/configs/imx_v7_android_defconfig |   36 ++++++++---------------------
 1 file changed, 10 insertions(+), 26 deletions(-)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index b3bbd23..5f81f65 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -336,7 +336,7 @@ CONFIG_IMX_HAVE_PLATFORM_SDHCI_ESDHC_IMX=y
 CONFIG_IMX_HAVE_PLATFORM_SPI_IMX=y
 CONFIG_MXC_REBOOT_ANDROID_CMD=y
 CONFIG_MXC_REBOOT_MFGMODE=y
-CONFIG_MACH_IMX_BLUETOOTH_RFKILL=y
+# CONFIG_MACH_IMX_BLUETOOTH_RFKILL is not set
 CONFIG_MX6_IR=y
 CONFIG_MX6_RAMOOPS=y
 # CONFIG_ARCH_OMAP2PLUS is not set
@@ -981,34 +981,15 @@ CONFIG_BT_HCIBTUSB=y
 CONFIG_BT_HCIUART=y
 CONFIG_BT_HCIUART_ATH3K=y
 CONFIG_BT_HCIVHCI=y
+CONFIG_BT_WILINK=y
+CONFIG_BT_HCIUART_LL=y
 
 CONFIG_FIB_RULES=y
 CONFIG_WIRELESS=y
 CONFIG_WEXT_CORE=y
 CONFIG_WEXT_PROC=y
-CONFIG_CFG80211=m
-# CONFIG_NL80211_TESTMODE is not set
-# CONFIG_CFG80211_DEVELOPER_WARNINGS is not set
-# CONFIG_CFG80211_REG_DEBUG is not set
-# CONFIG_CFG80211_CERTIFICATION_ONUS is not set
-CONFIG_CFG80211_DEFAULT_PS=y
-# CONFIG_CFG80211_DEBUGFS is not set
-# CONFIG_CFG80211_INTERNAL_REGDB is not set
-CONFIG_CFG80211_WEXT=y
-# CONFIG_LIB80211 is not set
-CONFIG_CFG80211_ALLOW_RECONNECT=y
-CONFIG_MAC80211=m
-CONFIG_MAC80211_HAS_RC=y
-# CONFIG_MAC80211_RC_PID is not set
-CONFIG_MAC80211_RC_MINSTREL=y
-CONFIG_MAC80211_RC_MINSTREL_HT=y
-CONFIG_MAC80211_RC_DEFAULT_MINSTREL=y
-CONFIG_MAC80211_RC_DEFAULT="minstrel_ht"
-# CONFIG_MAC80211_MESH is not set
-# CONFIG_MAC80211_LEDS is not set
-# CONFIG_MAC80211_DEBUGFS is not set
-# CONFIG_MAC80211_MESSAGE_TRACING is not set
-# CONFIG_MAC80211_DEBUG_MENU is not set
+# CONFIG_CFG80211 is not set
+# CONFIG_MAC80211 is not set
 # CONFIG_WIMAX is not set
 CONFIG_RFKILL=y
 CONFIG_RFKILL_PM=y
@@ -1177,7 +1158,7 @@ CONFIG_OF=y
 #
 # Device Tree and Open Firmware support
 #
-# CONFIG_PROC_DEVICETREE is not set
+CONFIG_PROC_DEVICETREE=y
 # CONFIG_OF_SELFTEST is not set
 CONFIG_OF_FLATTREE=y
 CONFIG_OF_EARLY_FLATTREE=y
@@ -1246,7 +1227,8 @@ CONFIG_EEPROM_AT25=y
 #
 # Texas Instruments shared transport line discipline
 #
-# CONFIG_TI_ST is not set
+CONFIG_TI_ST=y
+CONFIG_ST_HCI=y
 # CONFIG_SENSORS_LIS3_SPI is not set
 # CONFIG_SENSORS_LIS3_I2C is not set
 
@@ -3739,3 +3721,5 @@ CONFIG_AVERAGE=y
 # CONFIG_DDR is not set
 CONFIG_OID_REGISTRY=y
 # CONFIG_VIRTUALIZATION is not set
+#TI add for WL8
+CONFIG_IP_NF_TARGET_LOG=y
-- 
1.7.9.5

